build/Makefile

```diff
@@ -23,9 +23,9 @@ SCX_INTERMEDIATE_DIR := $(SCX_DIR)/intermediate/$(BUILD_CONFIGURATION)
 DSC_TARGET_DIR := $(DSC_DIR)/release
 
 RUBY_DIR := $(BASE_DIR)/source/ext/ruby
+RUBY_VER := 2.3.0
```

Comment:
@jeffaco ruby 2.3 has a new compiler which does not work with older BASERUBY versions such as 1.8. This was not a case in the past.